src/main/java/racing/Name.java

```diff
@@ -0,0 +1,29 @@
+package racing;
+
+import racing.inout.OutPutView;
+
+public class Name {
+    private final String name;
+
+    private Name(String name) {
+        this.name = name;
+    }
+
+    public static Name defaultOf(String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException("이름은 5자를 초과할 수 없습니다.");
```

Comment:
```suggestion
        if (name.length() > 5) {
            throw new IllegalArgumentException("이름은 5자를 초과할 수 없습니다.");
```

- 가독성을 위해 5에 대한 매직넘버 상수화해보기 
- 변하는 부분과 변하지 않는 부분을 생각해서 메시지 고민해보기
  - `이름의 길이를 초과했습니다.`
  - `이름은~ {변하는부분}~~ 없습니다 .`